projects/fusion-ui/components/input-inline/common/base/input-inline.base.component.ts

```diff
@@ -102,7 +111,19 @@ export abstract class InputInlineBaseComponent implements ControlValueAccessor,
             }
         });
 
-        this.isEditMode$.asObservable().pipe(takeUntil(this.onDestroy$)).subscribe(this.handleClickOutSideListener.bind(this));
+        this.isEditMode$
+            .asObservable()
+            .pipe(
+                takeUntil(this.onDestroy$),
+                filter(val => val)
+            )
+            .subscribe(this.handleClickOutSideListener.bind(this));
+
+        this.inputControl.valueChanges.pipe(takeUntil(this.onDestroy$)).subscribe(val => {
+            if (!!val && this.error) {
```

Comment:
move it to separate function 
like the above sunscribe

Reply:
Sure.